src/Microsoft.Azure.WebJobs.ServiceBus/Bindings/ServiceBusBinding.cs

```diff
@@ -41,7 +42,7 @@ public bool FromAttribute
             context.CancellationToken.ThrowIfCancellationRequested();
 
             string boundQueueName = _path.Bind(context.BindingData);
-            var messageSender = new MessageSender(_account.ConnectionString, boundQueueName);
+            var messageSender = _messagingProvider.CreateMessageSender(boundQueueName, _account.ConnectionString);
```

Comment:
Now routing creation of senders through the factory, as we do for receivers. This was an oversight when the factory was introduced originally.